Stripe/PublicHeaders/STPSourceCardDetails.h

```diff
@@ -25,7 +27,7 @@ typedef NS_ENUM(NSInteger, STPSourceCard3DSecureStatus) {
     STPSourceCard3DSecureStatusRequired,
 
     /**
-     3D Secure is optional. It is not required for successful charging,
+     3D Secure is optional. It is not required nor recommended for successful charging,
```

Comment:
I think this should be "or" but I'm not positive

Reply:
I'm not sure, I copied it from https://stripe.com/docs/sources/three-d-secure#check-requirement so gonna go with their grammar.